applications/CETF.md

```diff
@@ -0,0 +1,246 @@
+
+
+
+### Project Details
+
+We created first ETF on EOSIO called EOSETF, it consists of 13 EOS mainnet tokens. We deployed the fund to the mainnet in 2021 Q3. UI can accessed via https://eosetf.io/, contracts deployed to https://bloks.io/account/cet.f. Currently there are more than 150 investors holding EOSETF, TVL has peaked over 35k USD. The price of the EOSETF has been pegged to the price of the underlying tokens it consists of, since some users have created arbitrage bots to capture price discrepancies, and others just perfrom the arbitrage manually. At current stage (Phase 1) EOSETF is a proof of concept/MVP. We saw enough traction to start building the Phase 2.
+
+Phase 2 is being tested here: https://testing.eosetf.io/.
+
+New features in Phase 2:
+
+1. Automatic rebalancing
+
+In current Phase 1 it is manual and laborious process to rebalance the fund. The new rebalancing function enables easily to include and exclude tokens from the fund. EOSETF (and the future funds) will have fund managers that through https://polling.eosetf.io/ (currently connected to the testing fund, which you can try to rebalance. Pick tokens you'd like to include or exclude, click VOTE and then REBALANCE). There is no limit as to how many fund managers can participate in the rebalancing. The votes of fund managers are summed together to produce the final allocation. We've on-boarded three individuals from EOS community who have the skill and are interested in managing the EOSETF. Based on the votes of fund managers the REBALANCING function automatically exectues the trades via https://defibox.io/.
+
+2. CETF staking
+
+In Phase 1 we issued our native token CETF to all the creators of EOSETF (no token sale). In Phase 2 the CETF token can be staked. Staking enables to recieve portion of the fees that the EOSETF generates. Each week, stakers are able to claim EOSETF fees generated during the previous week.
+
+3. CETF mining
+
+Extremely important component of a funtioning ETF is its liquidity. As such we've created CETF mining function to incentivize liquidity providers for EOSETF/EOS pair on Defibox (https://defibox.io/pool-market-details/1232). Each week, stakers of LP tokens will be able to claim CETF tokens.
+
+
+  <br />
+  <br />
+Smart contracts of the above mentioned functions are 95% complete and currently in debugging phase. UI of the Phase 2 is 50% complete.
+
+Your support would help us to finish the Phase 2 and deploy it to EOS, WAX and TELOS. Each chain requires configuration since the rebalancing function needs to be configured with a DEX. Our plan is to on-board fund managers and custodians for the fund on WAX and Telos. The fund on EOS is already under msig between 5 Eden members (Vladislav Hramtsov, Lennar Lehestik, Chaney Moore, Dan Singjoy, Bastiaan Moossdorff).
+
+Phase 3 though is where the real magic happens. Once the Phase 2 is polished and perfected, the idea is to scale it in such way that anybody could easily set up their own ETFs.
```

Comment:
This seems to have answered my earlier question about creating new funds. But just to clarify further:

Is the idea that in Phase 2 it is possible for one to use the open source code to deploy their own fund if they wanted (with their own fund managers), but it requires deploying a copy of the contract into a new account? But in Phase 3, this functionality would be enabled through actions within the same contract?